File: scratch/test_login.py
import mysql.connector
import hashlib

# Connection configuration matches db.php & db_main.php
db_config = {
    'host': 'swift.herosite.pro',
    'user': 'surpriseville_partners',
    'password': 'Sv@123@4567',
    'database': 'surpriseville_partners'
}

vendor_email = 'nitish@gmail.com'
vendor_pass = '123456789'

print("=== VENDOR CHECK ===")
try:
    conn = mysql.connector.connect(**db_config)
    cursor = conn.cursor(dictionary=True)
    cursor.execute("SELECT id, name, email, password FROM vendors WHERE email = %s LIMIT 1", (vendor_email,))
    vendor = cursor.fetchone()
    if vendor:
        print(f"Found Vendor ID: {vendor['id']}, Name: {vendor['name']}")
        print(f"Password in DB: {vendor['password']}")
        md5_hash = hashlib.md5(vendor_pass.encode()).hexdigest()
        if vendor['password'] == md5_hash or vendor['password'] == vendor_pass:
            print("Password MATCH success!")
        else:
            print("Password MISMATCH!")
    else:
        print(f"Vendor not found by email: {vendor_email}")
        
    print("\n=== ADMIN CHECK ===")
    cursor.execute("SELECT id, username, password FROM admin WHERE username = 'admin' LIMIT 1")
    admin = cursor.fetchone()
    if admin:
        print(f"Found Admin ID: {admin['id']}, Username: {admin['username']}")
        print(f"Password in DB: {admin['password']}")
        md5_hash = hashlib.md5(b"123456789").hexdigest()
        if admin['password'] == md5_hash or admin['password'] == '123456789':
            print("Password MATCH success!")
        else:
            print("Password MISMATCH!")
    else:
        print("Admin not found")
        
    cursor.close()
    conn.close()
except Exception as e:
    print(f"Error: {e}")

File: scratch/test_login.php
<?php
// PDO Connection to partners database
try {
    $pdo_partners = new PDO("mysql:host=swift.herosite.pro;dbname=surpriseville_partners;charset=utf8mb4", "surpriseville_partners", "Sv@123@4567");
    $pdo_partners->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
} catch (PDOException $e) {
    die("Partners DB Connection failed: " . $e->getMessage() . "\n");
}

// Check Admin
$admin_user = 'admin';
$admin_pass = '123456789';

echo "\n=== ADMIN CHECK ===\n";
$stmt2 = $pdo_partners->prepare("SELECT id, username, password, status FROM admin_users WHERE username = ? LIMIT 1");
$stmt2->execute([$admin_user]);
$admin = $stmt2->fetch(PDO::FETCH_ASSOC);

if ($admin) {
    echo "Found Admin ID: " . $admin['id'] . ", Username: " . $admin['username'] . ", Status: " . $admin['status'] . "\n";
    echo "Password Hash in DB: " . $admin['password'] . "\n";
    if (password_verify($admin_pass, $admin['password'])) {
        echo "Password verification: MATCH (Valid bcrypt)!\n";
    } elseif ($admin['password'] === md5($admin_pass)) {
        echo "Password verification: MATCH (MD5)!\n";
    } elseif ($admin['password'] === $admin_pass) {
        echo "Password verification: MATCH (Plaintext)!\n";
    } else {
        echo "Password verification: MISMATCH!\n";
    }
} else {
    echo "Admin user not found in admin_users!\n";
}
